libbeat/autodiscover/builder/helper.go

```diff
@@ -73,14 +73,19 @@ func GetHintAsList(hints common.MapStr, key, config string) []string {
 
 // GetProcessors gets processor definitions from the hints and returns a list of configs as a MapStr
 func GetProcessors(hints common.MapStr, key string) []common.MapStr {
-	rawProcs := GetHintMapStr(hints, key, "processors")
-	if rawProcs == nil {
+	return GetConfigs(hints, key, "processors")
```

Comment:
Will `GetConfigs` used in other places in the future?

Reply:
yes. I plan to opensource an `emptydir` based builder that intends to use it. other uses for this would be for annotating the `fields` config.